src/tomography.jl

```diff
@@ -8,71 +8,69 @@ pulse_type: prepared states/meas. axes
 prep_meas: 1 for prep., 2 for meas. pulses
 """
 function tomo_gate_set(nbrQubits, nbrPulses; pulse_type="Clifford", prep_meas = 1)
-  id, sx, sy, sz = Matrix[complex(allpaulis(1)[i]) for i in 1:4]
-  gateSet = Dict()
-  Uset1Q = Dict{Int, Matrix{Complex128}}()
-  if nbrPulses==4
-    #Four pulse set
-    if pulse_type == "Clifford"
-      Uset1Q[1] = id
-      Uset1Q[2] = expm(-1im*(pi/4)*sx)
-      Uset1Q[3] = expm(-1im*(pi/4)*sy)
-      Uset1Q[4] = expm(-1im*(pi/2)*sx)
-    elseif pulse_type == "Tetra"
-      if prep_meas==1
-        Uset1Q[1] = id
-        Uset1Q[2] = expm(-1im*acos(-1/3)*sx)
-        Uset1Q[3] = expm(-1im*2*pi/3*sz)*expm(-1im*acos(-1/3)*sx)
-        Uset1Q[4] = expm(1im*2*pi/3*sz)*expm(-1im*acos(-1/3)*sx)
-      else
-        Uset1Q[1] = id
-        Uset1Q[2] = expm(1im*acos(-1/3)*sx)
-        Uset1Q[3] = expm(1im*acos(-1/3)*sx)*expm(1im*2*pi/3*sz)
-        Uset1Q[4] = expm(1im*acos(-1/3)*sx)*expm(-1im*2*pi/3*sz)
-      end
+    if nbrPulses==4
+        # Four pulse set
+        if pulse_type == "Clifford"
+            Uset1Q = [complex(RI),
```

Comment:
I think it gives the same answer, but `RI` is our notation for the Clifford group op representing the identity, while `Id` is the Pauli operator. Since everything else is a Clifford, might as well go Clifford all the way.